src/banner/banner.tsx

```diff
@@ -1,106 +1,204 @@
+            aria-labelledby={title ? titleId : descriptionId}
+            aria-describedby={title ? descriptionId : undefined}
             aria-live="polite"
             tabIndex={0}
-            borderRadius="standard"
-            className={[styles.banner, styles[`banner-${tone}`]]}
+            borderRadius="full"
+            className={styles.banner}
         >
-            <Columns space="medium" alignY="center">
-                <Column
-                    width="content"
-                    aria-hidden
-                    style={{
-                        /* Make sure the icon is centered vertically */
-                        lineHeight: 0,
-                    }}
-                >
-                    {icon}
-                </Column>
-                <Column>
-                    <Box paddingY="xsmall">
-                        {description ? (
-                            <Box id={titleId} className={[styles.title, styles[`title-${tone}`]]}>
-                                {title}
-                            </Box>
-                        ) : (
-                            <Box
-                                id={titleId}
-                                className={[
-                                    styles.title,
-                                    // If the banner does not have a description, we need to slightly tweak
-                                    // the styling of the title applying an extra css class
-                                    styles[`title-without-description`],
-                                    styles[`title-${tone}`],
-                                ]}
-                            >
+            {image ? <Box className={styles.image}>{image}</Box> : null}
+
+            <Box className={styles.content} display="flex" gap="small" alignItems="center">
+                <Box className={styles.staticContent} display="flex" gap="small" flexGrow={1}>
+                    <Box className={styles.icon}>
+                        {tone === 'neutral' ? icon : <BannerIcon tone={tone} />}
+                        {closeButton}
+                    </Box>
+
+                    <Box className={styles.copy} display="flex" flexDirection="column">
+                        {title ? (
+                            <Box id={titleId} className={styles.title}>
                                 {title}
                             </Box>
-                        )}
-                        {description ? (
-                            <Box
-                                id={descriptionId}
-                                className={[styles.description, styles[`description-${tone}`]]}
-                            >
-                                {description}
-                            </Box>
                         ) : null}
+                        <Box
+                            id={descriptionId}
+                            className={[styles.description, { [styles.secondary ?? '']: title }]}
```

Comment:
nit: I feel like `{ [styles.secondary ?? '']: title }` would be more readable as `title && styles.secondary` or `title ? styles.secondary : undefined`

Reply:
Changed to `title ? styles.secondary : null` 👍